helm/nostr/strfry-chart/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Default values for strfry-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: dockurr/strfry
+  tag: latest
+  pullPolicy: IfNotPresent
+
+persistence:
+  enabled: true
+  size: 1Gi
```

Comment:
is this all we need? or is this just for testing purposes?

Reply:
for testing, we can figure out what we need as we start using it?

Reply:
Kinda. I haven't needed to adjust a volume size yet but I don't think it's as simple as make this number bigger. I'm not too worried about it at the moment though. We can always redeploy if needed